app/client/templates/autoform/imageUpload.js

```diff
@@ -1,34 +1,35 @@
+var $hiddenInput,
```

Comment:
I think these vars should somehow be bound to the template but for now it can pass